### DISCUSSION TITLE
Version 1.2.4 release date and Empty address book

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk#6948] [General]
Hello,

this is the third time I have lost my address book. I don't understand why I suddenly find it empty. I seem to remember that others have had the same problem.

It would also be useful to have the possibility to export and import the address book.

I hope the problem is solved in version 1.2.4.

PDF

**Comment by rustdesk** [comment]
> export and import the address book.

@21pages

**Reply by pdifeo** [reply]
Thank you for your interest. When will the stable version 1.2.4 be released?

**Reply by rustdesk** [reply]
Maybe early March.

**Reply by pdifeo** [reply]
Any news about ver 1.2.4 and OV Certificate ?

**Reply by rustdesk** [reply]
The OV/EV have been solved, but now we are waiting for @Microsoft, https://github.com/rustdesk/rustdesk/discussions/6444#discussioncomment-8935775
